src/lib/pay-request/api_utils/ledger.js

```diff
@@ -32,9 +32,9 @@ const ledgerMethods = function ledgerMethods(instance) {
     const pageSize = 20
     const externalStatusMap = {
       all: [],
-      succeeded: [ 'success' ],
-      failed: [ 'declined', 'timedout', 'cancelled', 'error' ],
-      'in-progress': [ 'created', 'started', 'submitted', 'capturable' ]
+      succeeded: ['success'],
```

Comment:
Are these refactors automatic? I think this is working against the linter which encourages spaces, I'm not sure though. It would be good to keep things consistent whichever way it's done in the project (there may already be inconsistencies!)